HISTORY.rst

```diff
@@ -14,6 +14,22 @@ Release History
 
 * Updated both controlplane and dataplane SDKs to now use the newer 2021-06-30-preview API version.
 
+**IoT Central updates**
+
+* Added commands for Edge devices and modules:
+  - az iot central device edge module
+    - az iot central device edge module list
```

Comment:
Is there a way to add modules and manifests to edge devices? Or will this be part of a future update?

Reply:
this is planned for our API May release. will add to CLI at the same time.